architecture/Decisions.md

```diff
@@ -0,0 +1,53 @@
+
+## How Decisions Work
+
+The Decisions feature can be broken up into three main views.
+
+1. The Decisions List
+2. The Decision Dialog and Preview
+3. The Decision details
+
+The Decisions List, available at `/:colony/decisions`, lists all the decisions that have occurred in that Colony. Much like Motions, a Decision requires a 10% stake to appear in the Decisions list. The Decisions list can be filtered by the domain the Decision was created in. Clicking an item in the Decisions list will take the user to the Decision details page, with the exception of the `DraftDecisionItem`, which shows the user if they currently have a draft saved locally.
+
+The Decision Dialog is a rich text input form that allows users to supply their Decision with a title and a description. It will appear when clicking "New Decision". Submitting this form takes the user to a Decision Preview, where they can review, edit, and delete their draft. Once they're happy, they can click "Publish", which will take them to the Decision details page.
+
+Decision details looks mostly the same as for motions, and indeed, a Decision follows the same lifecycle as a motion, with the exception of Finalization (since a Decision doesn't trigger an on-chain action, there's no need to finalize it). The main difference in the UI is the rendering of the decision description as rich text. 
+
+### Decision Drafts
+
+A draft decision is what's initially created when a user submits the Decision Dialog form. This draft needs to be available at `/:colony/decisions/preview`, so a user can preview their draft.
+
+We also need access to it in the Decisions list, since we display a `DraftDecisionItem` there to show the user they have a draft saved locally. 
+
+Drafts are stored in local storage so they persist between page reloads. We also store decision details in Redux, which enables us to re-render any component using these details whenever they change (such as when editing / deleting a draft). When the app loads, we use `ColonyDecisionContext` to take the decision from local storage and populate it in Redux. We then read the decision directly from redux, which gives us access to the decision data globally, which is handy for accessing it from different screens and also from Dialogs. 
+
+### Decision saga
+
+Once a user is happy with their Decision, they can publish it. 
+
+This fires the `MOTION_CREATE_DECISION` redux action, which triggers the `createDecisionMotion` saga.
+
+In this saga, we trigger the `createMotion` action on the `VotingReputation` contract. 
+
+We then store the Decision details in the database, under the `ColonyDecision` model. We store the Decision id in the format: `<colonyAddress>_decision_<txHash>`, so it can be easily retreived from the block ingestor.
```

Comment:
Would it be worth mentioning the delete saga? The heading suggests there's only one decision related saga